### PR TITLE
chore: Include dev mode verbose flag

### PR DESCRIPTION
**Pull request by Danwakeem**
Description
Included a `--verbose` option for dev mode. I was using this a bit over the weekend and thought it would be nice to have an option to get more information out the req and response as well as the input/output of spans 🙏

### CODE REVIEW

File: lib/cli/commands-schema/no-service.js [hunk 1]
```diff
@@ -39,6 +39,12 @@ commands.set('', {
       type: 'string',
       shortcut: 'f',
     },
+    'verbose': {
+      usage:
+        'Include more dev mode information including the body of requests and responses as well as input and output of incoming spans.',
+      type: 'string',
+      shortcut: 'v',
+    },
```
Comment (medikoo) [inline lib/cli/commands-schema/no-service.js:47]: This option is already defined in common options set: https://github.com/serverless/serverless/blob/a2d777ae8303fb96e35caf3754435e3742a7c286/lib/cli/commands-schema/common-options/global.js#L6

It serves our logging, so with this flag we produce more verbose output. I think you can just take advantage of it in dev mode as well

File: lib/cli/interactive-setup/console-dev-mode-feed.js [hunk 2]
```diff
@@ -225,7 +243,7 @@ module.exports = {
   },
 
   async run(context) {
-    streamBuff.on('data', handleSocketMessage);
+    streamBuff.on('data', handleSocketMessage(context.options.verbose));
```
Comment (medikoo) [inline lib/cli/interactive-setup/console-dev-mode-feed.js:246]: It'll be probably more future proof to pass context here.

**Review by medikoo — CHANGES_REQUESTED** [review]
Thanks @Danwakeem. Good call, still see my comments

File: lib/cli/commands-schema/no-service.js (hunk 1 above)
Comment (Danwakeem) [reply lib/cli/commands-schema/no-service.js:47]: Ahh okay nice thank you for pointing this out 👍

**Review by medikoo — APPROVED** [review]
👍

**Pull request merged**